CMakeLists.txt

```diff
@@ -170,6 +170,12 @@ target_link_libraries(srds_augeas ${AUGEAS_LIBRARIES})
 target_link_libraries(ay_startup ${AUGEAS_LIBRARIES})
 include_directories(${AUGEAS_INCLUDE_DIRS})
 
+# libxml2
+find_package(LibXML2 REQUIRED)
```

Comment:
There is a standard module for this called `FindLibXml2.cmake`, no need to write your own.